src/mainboard/pcengines/apu2/variants/apu4/devicetree.cb

```diff
@@ -64,7 +64,9 @@ chip northbridge/amd/pi/00730F01/root_complex
 						io 0x60 = 0x2e8
 						irq 0x70 = 3
 					end
-					device pnp 2e.8 off end
+					device pnp 2e.8 on
+						io 0x60 = 0x220
```

Comment:
@piotr-kleins apu2 0x290 bu apu4 0x220? i do not understand

Reply:
@miczyg1 As it is WIP, I test it on apu4. That's why I change only that configuration. When everything works I will change another apu variants also.